sim/ngspice/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set NGSPICE_TAG = GIT_DESCRIBE_TAG|replace('ngspice-', '') %}
+{% set version = '%s_%04i_%s'|format(NGSPICE_TAG, GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: ngspice
+  version: {{ version }}
+
+source:
+   git_url: git://git.code.sf.net/p/ngspice/ngspice
+   git_rev: master
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - CONDA_OUT
+    - TEST_PACKAGE
```

Comment:
```suggestion
```

I know these are present in most of the recipes but they are only needed if you want to pass such variables to `build.sh`. Since neither the script nor ngspice's build scripts use them, let's remove them.